figures/opening-virtual-channel.tex

```diff
@@ -13,8 +13,8 @@
 
 
       \pic (G) at (0, -2.5) {guarantee};
-      \node[circB] (b2) at (-1,-4) {};
-      \node[circA] (a2) at (0,-4) {};
+      \node[circA] (a2) at (-1,-4) {};
+      \node[circB] (b2) at (0,-4) {};
```

Comment:
The changes to this file only affect Figure 25. The change consists of permuting the positions of a2 and b2, as well as the edger connecting them with the joint channel. This happens in 3 of the 4 frames of the figure.